### DISCUSSION TITLE
chia plots check gives Runtime Error but plot log shows no warnings

### DISCUSSION THREAD

**Discussion by maxrozenblum2020** [Chia-Network/chia-blockchain#6273] [Support]
Is it possible that the `chia plots check` method has a bug? I have 2 plots that show no warning messages in their plot logs and appeared to finish all the print statements that a normal plot log.

plot log finishes normally
![image](https://user-images.githubusercontent.com/60197354/120217982-1ce10280-c207-11eb-84b0-b7a58f3b8cfe.png)

`chia plots check` gets a runtime error?
![image](https://user-images.githubusercontent.com/60197354/120218073-371ae080-c207-11eb-9ec3-3690dc2fbe1a.png)

Is this plot viable? How can I check this if chia plots check may have a bug? Or should I cut my losses and delete these two.

**Comment by tiny0little** [comment]
make sure you don't have error in os logs via `dmesg`
try to move the plot to different drive, if you have one

similar problem - https://github.com/Chia-Network/chia-blockchain/issues/5895

**Comment by room101-dev** [comment]
> Is it possible that the `chia plots check` method has a bug? I have 2 plots that show no warning messages in their plot logs and appeared to finish all the print statements that a normal plot log.
> 
> plot log finishes normally
> ![image](https://user-images.githubusercontent.com/60197354/120217982-1ce10280-c207-11eb-84b0-b7a58f3b8cfe.png)
> 
> `chia plots check` gets a runtime error?
> ![image](https://user-images.githubusercontent.com/60197354/120218073-371ae080-c207-11eb-9ec3-3690dc2fbe1a.png)
> 
> Is this plot viable? How can I check this if chia plots check may have a bug? Or should I cut my losses and delete these two.

U MUST enter the log level in the config.yaml, to 'INFO' or higher, by default as about two versions ago CHIA-DEV-TEAM turned off full logging information.

**Reply by maxrozenblum2020** [reply]
I already did this long ago.. This has nothing to do with the problem I am describing.

**Comment by uldh** [comment]
i have the same problem, does anyone know why "chia plots check" report those "ERROR" ? i have 1000 plots ERROR reported, but those 1000 plots works well now.

**Comment by BrandtH22** [comment]
@maxrozenblum2020 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
